<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>操作</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="horizontalWidget_2" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QCheckBox" name="cbSound">
                 <property name="styleSheet">
                  <string notr="true">
QCheckBox::indicator { 
    width: 50px;
    height: 50px;
}

QCheckBox::indicator::unchecked {   
    image: url(:/my/res/checkbox_unchecked.png);
}

QCheckBox::indicator::checked { 
    image: url(:/my/res/checkbox_checked.png);
}</string>
                 </property>
                 <property name="text">
                  <string>开启音效</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cbShowOrderNum">
                 <property name="styleSheet">
                  <string notr="true">
QCheckBox::indicator { 
    width: 50px;
    height: 50px;
}

QCheckBox::indicator::unchecked {   
    image: url(:/my/res/checkbox_unchecked.png);
}

QCheckBox::indicator::checked { 
    image: url(:/my/res/checkbox_checked.png);
}</string>
                 </property>
                 <property name="text">
                  <string>显示顺序</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdNumber">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 240, 70);</string>
               </property>
               <property name="smallDecimalPoint">
                <bool>false</bool>
               </property>
               <property name="digitCount">
                <number>10</number>
               </property>
               <property name="value" stdset="0">
                <double>0.000000000000000</double>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnNewGame">
               <property name="styleSheet">
                <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 24px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}
</string>
               </property>
               <property name="text">
                <string>新游戏</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRegretChess">
               <property name="styleSheet">
                <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 24px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
               </property>
               <property name="text">
                <string>悔棋</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>服务端/客户端信息</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTextEdit" name="txtMsg">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">
font-size: 20px;
font-weight:bold;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Titillium Web'; font-size:20px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="txtSendMsg">
            <property name="styleSheet">
             <string notr="true">border: 5px bold   rgb(74, 255, 116);
font-size: 20px;
font-weight:bold;
padding: 5px 5px;</string>
            </property>
            <property name="text">
             <string>发送消息</string>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
            <property name="cursorPosition">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSendMsg">
            <property name="styleSheet">
             <string notr="true">    QPushButton {
      padding: 5px 10px;
      font-size: 17px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 5px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>IP地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="txtIP">
              <property name="styleSheet">
               <string notr="true">border: 5px bold   rgb(74, 255, 116);
font-size: 20px;
font-weight:bold;</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentText">
               <string>127.0.0.1</string>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
              <property name="maxCount">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>端口号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtPort">
              <property name="toolTip">
               <string>端口号</string>
              </property>
              <property name="styleSheet">
               <string notr="true">
border: 5px bold   rgb(74, 255, 116);
font-size: 20px;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>6666</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="cursorMoveStyle">
               <enum>Qt::LogicalMoveStyle</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QRadioButton" name="radioLAN">
            <property name="styleSheet">
             <string notr="true">
QRadioButton::indicator { 
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
    image: url(:/my/res/radio_unchecked.png);
}

QRadioButton::indicator::checked { 
    image: url(:/my/res/radio_checked.png);
}

QRadioButton ,QCheckBox{
    spacing: 5px;
    font-size: 15px;
}
</string>
            </property>
            <property name="text">
             <string>局域网</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioWAN">
            <property name="styleSheet">
             <string notr="true">
QRadioButton::indicator { 
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
    image: url(:/my/res/radio_unchecked.png);
}

QRadioButton::indicator::checked { 
    image: url(:/my/res/radio_checked.png);
}

QRadioButton ,QCheckBox{
    spacing: 5px;
    font-size: 15px;
}
</string>
            </property>
            <property name="text">
             <string>远程服务器</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="btnServerListen">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 17px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
            </property>
            <property name="text">
             <string>监听</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClientConn">
            <property name="styleSheet">
             <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 17px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
            </property>
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRestartGame">
            <property name="styleSheet">
             <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 17px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
            </property>
            <property name="text">
             <string>重新开始</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuitConn">
            <property name="styleSheet">
             <string notr="true">    QPushButton {
      padding: 15px 25px;
      font-size: 17px;
      text-align: center;
      text-decoration: none;
      outline: none;
      color: #fff;
      background-color: #4CAF50;
      border: none;
      border-radius: 10px;
    }

QPushButton:hover {
      background-color: #3e8e41
}

QPushButton:disabled{
	background-color: rgb(34, 80, 36);
}
QPushButton:pressed{
	background-color: rgb(45, 104, 47);
}</string>
            </property>
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>先手模式</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="personFirst">
            <property name="styleSheet">
             <string notr="true">
QRadioButton::indicator { 
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
    image: url(:/my/res/radio_unchecked.png);
}

QRadioButton::indicator::checked { 
    image: url(:/my/res/radio_checked.png);
}

QRadioButton ,QCheckBox{
    spacing: 5px;
    font-size: 15px;
}
</string>
            </property>
            <property name="text">
             <string>人类先手</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="robotFirst">
            <property name="styleSheet">
             <string notr="true">
QRadioButton::indicator { 
    width: 25px;
    height: 25px;
}

QRadioButton::indicator::unchecked {
    image: url(:/my/res/radio_unchecked.png);
}

QRadioButton::indicator::checked { 
    image: url(:/my/res/radio_checked.png);
}

QRadioButton ,QCheckBox{
    spacing: 5px;
    font-size: 15px;
}
</string>
            </property>
            <property name="text">
             <string>机器先手</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtSendMsg</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSendMsg</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtSendMsg</sender>
   <signal>returnPressed()</signal>
   <receiver>txtSendMsg</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
